{"success": false, "error": "Traceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 544, in _run_solc\n    ret: Dict = json.loads(stdout)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/__init__.py\", line 357, in loads\n    return _default_decoder.decode(s)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 337, in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 355, in raw_decode\n    raise JSONDecodeError(\"Expecting value\", s, err.value) from None\njson.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 826, in main_impl\n    ) = process_all(filename, args, detector_classes, printer_classes)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 86, in process_all\n    compilations = compile_all(target, **vars(args))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 637, in compile_all\n    compilations.append(CryticCompile(target, **kwargs))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 117, in __init__\n    self._compile(**kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 548, in _compile\n    self._platform.compile(self, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 154, in compile\n    targets_json = _get_targets_json(compilation_unit, self._target, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 276, in _get_targets_json\n    return _run_solc(\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 548, in _run_solc\n    raise InvalidCompilation(f\"Invalid solc compilation {stderr}\")\ncrytic_compile.platform.exceptions.InvalidCompilation: Invalid solc compilation Error: Multiple SPDX license identifiers found in source file. Use \"AND\" or \"OR\" to combine multiple licenses. Please see https://spdx.org for more information.\n--> /Users/giacomoibba/Desktop/mainnet/2a/2aaA9Ae7101e0A59C52916D8db9acda311b1c296_veFXSYieldDistributorV3.sol\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/Math/Math.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/mainnet/2a/2aaA9Ae7101e0A59C52916D8db9acda311b1c296_veFXSYieldDistributorV3.sol:29:1:\n   |\n29 | import \"../Math/Math.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/Math/SafeMath.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/mainnet/2a/2aaA9Ae7101e0A59C52916D8db9acda311b1c296_veFXSYieldDistributorV3.sol:30:1:\n   |\n30 | import \"../Math/SafeMath.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/Curve/IveFXS.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/mainnet/2a/2aaA9Ae7101e0A59C52916D8db9acda311b1c296_veFXSYieldDistributorV3.sol:31:1:\n   |\n31 | import \"../Curve/IveFXS.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/Uniswap/TransferHelper.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/mainnet/2a/2aaA9Ae7101e0A59C52916D8db9acda311b1c296_veFXSYieldDistributorV3.sol:32:1:\n   |\n32 | import \"../Uniswap/TransferHelper.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/ERC20/ERC20.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/mainnet/2a/2aaA9Ae7101e0A59C52916D8db9acda311b1c296_veFXSYieldDistributorV3.sol:33:1:\n   |\n33 | import \"../ERC20/ERC20.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/ERC20/SafeERC20.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/mainnet/2a/2aaA9Ae7101e0A59C52916D8db9acda311b1c296_veFXSYieldDistributorV3.sol:34:1:\n   |\n34 | import \"../ERC20/SafeERC20.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/Utils/ReentrancyGuard.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/mainnet/2a/2aaA9Ae7101e0A59C52916D8db9acda311b1c296_veFXSYieldDistributorV3.sol:35:1:\n   |\n35 | import \"../Utils/ReentrancyGuard.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/2a/Owned.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/mainnet/2a/2aaA9Ae7101e0A59C52916D8db9acda311b1c296_veFXSYieldDistributorV3.sol:36:1:\n   |\n36 | import \"./Owned.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/Common/Context.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/2a/2aaA9Ae7101e0A59C52916D8db9acda311b1c296_veFXSYieldDistributorV3.sol:611:1:\n    |\n611 | import \"../Common/Context.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/2a/IERC20.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/2a/2aaA9Ae7101e0A59C52916D8db9acda311b1c296_veFXSYieldDistributorV3.sol:612:1:\n    |\n612 | import \"./IERC20.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/Math/SafeMath.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/2a/2aaA9Ae7101e0A59C52916D8db9acda311b1c296_veFXSYieldDistributorV3.sol:613:1:\n    |\n613 | import \"../Math/SafeMath.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/Utils/Address.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/2a/2aaA9Ae7101e0A59C52916D8db9acda311b1c296_veFXSYieldDistributorV3.sol:614:1:\n    |\n614 | import \"../Utils/Address.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/2a/IERC20.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/2a/2aaA9Ae7101e0A59C52916D8db9acda311b1c296_veFXSYieldDistributorV3.sol:947:1:\n    |\n947 | import \"./IERC20.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/Math/SafeMath.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/2a/2aaA9Ae7101e0A59C52916D8db9acda311b1c296_veFXSYieldDistributorV3.sol:948:1:\n    |\n948 | import \"../Math/SafeMath.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/Utils/Address.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/2a/2aaA9Ae7101e0A59C52916D8db9acda311b1c296_veFXSYieldDistributorV3.sol:949:1:\n    |\n949 | import \"../Utils/Address.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/Common/Context.sol\" not found: File not found.\n    --> /Users/giacomoibba/Desktop/mainnet/2a/2aaA9Ae7101e0A59C52916D8db9acda311b1c296_veFXSYieldDistributorV3.sol:1143:1:\n     |\n1143 | import \"../Common/Context.sol\";\n     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/Math/SafeMath.sol\" not found: File not found.\n    --> /Users/giacomoibba/Desktop/mainnet/2a/2aaA9Ae7101e0A59C52916D8db9acda311b1c296_veFXSYieldDistributorV3.sol:1144:1:\n     |\n1144 | import \"../Math/SafeMath.sol\";\n     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\n\n", "results": {}}
